### DISCUSSION TITLE
A Profile Save To Yml

### DISCUSSION THREAD

**Discussion by ABIGDachshunch** [diamond-rip/Eden#107] [General]
I know this will mess up something but please try once to test!

**Comment by GoodestEnglish** [comment]
No point to use yml to save profile if it will break things and you can find free mongo host easily.

**Reply by ABIGDachshunch** [reply]
yes there is a point because using mongo db is just too hard and so much bugs and lag that was happen w eden like when I just did disconnect and reconnect it'll automatic send me to loading your profile mode and I have to redo everything please....

**Reply by ABIGDachshunch** [reply]
please try!

**Reply by ABIGDachshunch** [reply]
and mongodb break things more...

**Reply by GoodestEnglish** [reply]
If the case you mentioned is true, you should open an issue. Anyways, won't change how profile saving right now.
